app/serializers/user.js

```diff
@@ -0,0 +1,30 @@
+// app/serializers/user.js
+import JSONAPISerializer from '@ember-data/serializer/json-api';
+
+export default class UserSerializer extends JSONAPISerializer {
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
```

Comment:
This is fine, although I think if you need just simple mapping one property into another we can use property `attrs` in the serializer, here reference: 
https://api.emberjs.com/ember-data/5.3/classes/JSONSerializer/properties/attrs?anchor=attrs